dev/release/utils-update-docs-versions.py

```diff
@@ -26,6 +26,7 @@
 
 main_versions_path = dir_path + "/docs/source/_static/versions.json"
 r_versions_path = dir_path + "/r/pkgdown/assets/versions.json"
+r_html_path = dir_path + "/r/pkgdown/assets/versions.html"
```

Comment:
Could you add a generated HTML to this pull request?
We should version control the HTML, right?